polynote-kernel/src/main/scala/polynote/config/PolynoteConfig.scala

```diff
@@ -30,6 +33,24 @@ object Mount {
   implicit val decoder: Decoder[Mount] = deriveConfigDecoder[Mount]
 }
 
+
+
+final case class Kernel(
+  listen: String = "127.0.0.1",
+  portRange: Range = Range.inclusive(1234,4321)
```

Comment:
Can this be an `Option[Range]` instead of having this default? Let's also make `listen` optional, in case the user doesn't want to force a particular interface but instead use the system default interface.

Reply:
Ok I will try.